sass/layout/_navigation.scss

```diff
@@ -201,4 +201,8 @@
     }
 }
 
-
+.app__body {
+    .navbar-right__icon {
+        background: rgba(var(--sidebar-header-text-color-rgb), 0.2);
```

Comment:
let's move this up under an existing selector on https://github.com/mattermost/mattermost-webapp/pull/6745/files#diff-33df6171c54ae8787ddb1c33f38d4496R91.

Reply:
@ali-farooq0 thanks for the comment! Would you be able to clarify what you mean by an existing selector? Do you mean putting it under here?
https://github.com/mattermost/mattermost-webapp/blob/ff6dcbaa312c1aa8f5de57b0672fc8f8dd083f86/sass/layout/_navigation.scss#L90

Reply:
Yes. To avoid creating this new section

Reply:
Ok, done!